recipes/stages/classification/warmstart.yaml

```diff
@@ -0,0 +1,45 @@
+_base_: [
+    # remove default.py to disenable task_adapt
+    '../_base_/dist/dist.py',
+    '../_base_/logs/tensorboard_logger.py',
+    '../_base_/optimizers/lars.py',
+    '../_base_/runners/iter_runner.py',
+    '../_base_/schedules/cos_anneal.py',
+    '../_base_/data/warmstart_cls_data.py',
+]
+
+# in default.py settings
+cudnn_benchmark: true
```

Comment:
I think cudnn and deterministic is set for experimental purpose, but this should follow other recipes

Reply:
I'll fix it.

Reply:
Actually, I wanted to avoid `default.py` because I don't want to use `task_adapt` in warmstart, so I removed `default.py` in `_base_` and added others in `default.py` except for `task_adapt`.
In `default.py`, `cudnn_benchmark` is set to `True` as default setting, and it seems not for experimental purpose unlike you said. (Of course, I agree `deterministic` and it will be set to `False`.)

Reply:
If cudnn_benchmark is True in default.py then here, it should be True